main/lsp/lsp.h

```diff
@@ -53,59 +37,6 @@ class LSPLoop {
     /** ID of the main thread, which actually processes LSP requests and performs typechecking. */
     std::thread::id mainThreadId;
 
-    void addLocIfExists(const core::GlobalState &gs, std::vector<std::unique_ptr<Location>> &locs, core::Loc loc) const;
```

Comment:
`handle*` are now moved to specific `LSPTask` subclasses' `runRequest` methods.

The other methods are moved to `LSPTask.h`.